### ISSUE TITLE
Presenting community updates to newcomers

### ISSUE THREAD

**Issue by kripken** [WebAssembly/design#1519]
In a discussion on Bluesky today I was surprised to hear that the wasm community can look "inactive" or opaque. That seemed odd to me at first since we do so much work on github and we publish meeting notes, etc. etc. but as I listened I realized that we can do better. Here are some concrete points that are maybe worth thinking about:

* The [wasm CG page](https://www.w3.org/community/webassembly/) does not look active. The last post is from 2015 ("Call for Participation in WebAssembly Community Group"). This may be the first page a newcomer sees, so that impression matters.
* The first two CG mailing lists linked to from there ([this](https://lists.w3.org/Archives/Public/public-webassembly/) and [this](https://lists.w3.org/Archives/Public/public-webassembly-announce/)) have a post or two a month.
* The wasm CG page links to the [WebAssembly github org](https://github.com/webassembly), which is great, but it's a tiny link with no explanation. That is, it is not obvious that that link goes to where 99% of activity is happening!
* The webassembly.org site has lots of great content, but when it comes to the [community page](https://webassembly.org/community/resources/), we link to discord and a few others but not to the github org, so it's missing out on where most design discussions happen.

Overall, for someone that knows a bit about wasm and wants to see what the community is working on right now, the above points make it hard to find that out, I think.

Perhaps a message like this could help, on the w3c CG page and the github org page?

> Active discussions on WebAssembly's evolution are happening primarily in the WebAssembly GitHub organization. You can follow the `design` repo for general discussions, and for specific proposals, see the `proposals` repo and click through to the ones that interest you. There is also a main bi-weekly meeting whose meeting notes are published, and specific meetings for the specific proposals.

I think that covers most design activity? Mentioning that could maybe help guide newcomers find the discussions that interest them.

Another thing that might help, but would be more work, is some kind of main "activity feed" for proposals. Items mentioned there could be: "Wasm GC goes to phase 4"; "Memory64 goes to phase 3"; maybe also mentions of talks in the bi-weekly meeting; etc. That could give people an idea of ongoing activity.

cc @tomayac who may have been thinking on related matters recently.

**Comment by tomayac** [comment]
This is great feedback and I have added this to my list of things to think about for the site redesign (internally at Google, this is b/349534136).

**Comment by dschuff** [comment]
The CG landing page (https://www.w3.org/community/webassembly/) is powered by wordpress and it looks like I can just post things there. Maybe we should make a post (which would presumably replace the one from 2015 on the landing page) that would give the general information that all of the interesting stuff is happening on github, and if you want to see what the CG is doing, look there.

**Comment by dschuff** [comment]
I updated https://www.w3.org/community/webassembly/ with a generic info post pointing people to the github resources. Feedback welcome.

**Comment by kripken** [comment]
@dschuff Looks great, thanks for updating it!

**Comment by sunfishcode** [comment]
@kripken With the updates above, can this issue now be closed?

**Comment by dschuff** [comment]
Yes, I think so. I think the update addresses most of what's in OP except maybe about webassembly.org. But we can file issues (or probably there are existing ones) in https://github.com/WebAssembly/website for those.
